newsrevenuehub/rev-engine
# [NRH-276] Styles edit

DON’T MERGE ME IN TO DEVELOP UNTIL:
- [x] We've culled staging of any overly verbose style names (>50chars)

#### What's this PR do?
Adds a org admin interface for editing styles. This required an interface for view styles. Logically, this work includes the ability to add styles from somewhere other than the page editor.

We've also remove "Overview" here and changed the "Page" tab to "Content". The content tab now contains pages and styles. Each of those larger sections is collapsible.

#### How should this be manually tested?
Go to the new "content" tab.
Collapse those pages.
Notice the styles list. Click on a style to edit it.
Click on the plus button to add a new style.
Notice the green icon indicating that a style is used on a live page and that editing it might be trouble.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
